### ISSUE TITLE
REDAP Team Review Handbook - AH Updates

### ISSUE THREAD

**Issue by AHolder1** [CAWaterBoardDataCenter/equity-data-handbook#33]
After @AHolder1 is done with initial review and adding content to each page of the Handbook - other REDAP Team Co-working members follow and review/revise content as needed in preparation for internal review

- [ ] Welcome
- [ ] Background
- [ ] Getting Started
- [ ] Best Practices
- [ ] Establishing Common Language
- [ ] Planning
- [ ] Data Preparation
- [ ] Data Collection
- [ ] Surveys
- [ ] Data Processing (create page and go over tidying with an equity lens)
- [ ] Data Analysis
- [ ] Data Visualization
- [ ] Demographics Data
- [ ] CalEnviorScreen
- [ ] Turning Data to Info
- [ ] Preservation & Storage
- [ ] Data Sharing
- [ ] Evaluation
- [ ] Documentation
- [ ] SWAMP Use Case
- [ ] Demographics Use Case
- [ ] Resources (if page is getting too long - consider creating a resource section and adding pages for each resource type)
- [ ] Inspiration
- [ ] Add AI Page somewhere (collect and process? Use case?)

**Comment by NMartorano** [comment]
We should explore if it's wise to have so much on the best practices subpage and then also have other categories with the same names and headers, I think that could get confusing but I'm not sure...

**Comment by NMartorano** [comment]
Data Collection page:  Should Surveys be moved somewhere else?

**Comment by esuglia** [comment]
Under Data Collection > Common Data Sources there is a header for Locally and Regionally Collected Data with no links. Do we want to add some or remove that section?

**Comment by AHolder1** [comment]
Hi @esuglia - FYI the Data collection page isn't yet ready for your review. I still have a number of edits I plan on making that aren't reflected in the page yet.

**Comment by esuglia** [comment]
Oh thanks @AHolder1 ! I missed the "to do list" at the bottom.

**Comment by esuglia** [comment]
On the "Quality Assurance" page, I am confused by this sentence: "Do the standards, thresholds, or limits we’re using align with the needs of those facing inequity or disparities - **are or likely will be affected by a policy, program, or process**? If not, how might we modify the them to better align with those needs?" specifically the part of the sentence I bolded after the dash. Can you rephrase to clarify, or maybe separate into two sentences?

**Comment by NMartorano** [comment]
Hmm that seems like two thoughts that got combined.  The bolded sentence should be removed and maybe reframed as a separate question although I'm not sure how that fits into Quality Assurance.  In my opinion the question of potential impacts to underserved communities should be asked at the onset of any Policy, Program and process, or at least that's a goal.

Nick Martorano
He/Him/His
Assistant Deputy Director, Surface Water/Regulatory Branch
Division of Water Quality
State Water Resources Control Board
1001 I Street, Office 15th Floor, Sacramento, CA 95814

Phone:      Office: (916) 341-5514 (voicemail to email)
Email:       ***@***.******@***.***>
Website:   Division of Water Quality Webpage<https://www.waterboards.ca.gov/water_issues/programs/water_quality/>

I respectfully acknowledge that we live and work on traditional, ancestral & unceded land of the Miwok<https://native-land.ca/maps/territories/plains-miwok/>, Patwin<https://native-land.ca/maps/territories/patwin/> and Nisenan<https://native-land.ca/maps/territories/nisenan/> nation. Whose land are you on?<https://urldefense.com/v3/__https://native-land.ca/__;!!Mih3wA!RY4Qv3WLTYK9Pr9FmJhtQbmjuGaYX_x5KT6bZtSxcFFQd1sgvBebjN8K3A6gAhVu$>

Learn about CalEPA's Commitment to Racial Equity<https://calepa.ca.gov/about/calepa-racial-equity-home/> and why we lead with race.
***@***.***[logo][Right click to save image]

"CONFIDENTIALITY NOTICE: This Communication, with its contents, may contain confidential and/or legally privileged information. It is solely for the use of the intended recipient(s). Unauthorized interception, review, use or disclosure is prohibited and may violate applicable laws, including the Electronic Communications Privacy Act. If you are not the intended recipient, please contact the sender and destroy all copies of the communication."
